pennylane/pulse/convenience_functions.py

```diff
@@ -21,32 +22,45 @@
 except ImportError:
     has_jax = False
 
-
-def constant(windows: List[Tuple[float]] = None):
-    """Returns a callable ``f(p, t)`` that returns ``p`` inside the time
-    windows defined in ``windows``.
+# pylint: disable=unused-argument
+def constant(scalar, time):
+    """Parametrized function that is constant in time and returns the given ``scalar``.
 
     Args:
-        windows (Tuple[float, Tuple[float]]): List of tuples containing time windows where
-        ``f(p, t)`` is evaluated. If ``None``, it is always evaluated. Defaults to ``None``.
+        scalar (float): the scalar to be returned
+        time (float): Time. This argument is not used.
+
+    This function is mainly used to build a parametrized hamiltonian that can be differentiated
+    with respect to its constant term.
 
     **Example**
 
     The ``constant`` function can be used to create a parametrized hamiltonian
 
-    >>> windows = [(1, 7), (9, 14)]
-    >>> H = qml.pulse.constant(windows) * qml.PauliX(0)
+    >>> H = qml.pulse.constant * qml.PauliX(0)
 
-    When calling the parametrized hamiltonian, ``constant`` will return the input parameter only
-    when the time is inside the given time windows
+    When calling the parametrized hamiltonian, ``constant`` will always return the input parameter
 
     >>> params = [5]
```

Comment:
Wait, this does not work, does it? `constant` just returns the list, which is not a scalar... Or does this work anyways, unwrapping the single parameter in `[5]`?

Reply:
The input to the `ParametrizedHamiltonian` class should always be a `list` containing the parameters for each callable.

Reply:
Maybe we should make this clearer in the documentation

Reply:
Ah, yes of course. Sorry I was thinking too much about the single function..
Being more clear never hurts but I think this one is on me :D